pages/feed/index.tsx

```diff
@@ -101,10 +101,11 @@ const Feeds: WithPageLayout<HighlightSSRProps> = (props: HighlightSSRProps) => {
   useEffect(() => {
     const queryParams = new URLSearchParams(window.location.search);
     const newHighlight = queryParams.get("new");
+    const prURL = queryParams.get("prurl");
```

Comment:
This appears to not be used anywhere? If so, it can be removed.

Reply:
Thanks, just removed it. not sure how I did not notice that lol